dask_ml/cluster/k_means.py

```diff
@@ -22,6 +21,9 @@
 from ..utils import _timed, _timer, check_array, row_norms
 from ._compat import _k_init
 
+import numba  # isort:skip (see https://github.com/dask/dask-ml/pull/577)
```

Comment:
The import order here was apparently important. Do you know why it was moved?

Reply:
Yeah. I will remove this change.